config/config.go

```diff
@@ -196,10 +201,23 @@ func Validate() error {
 	if maxMessageSize <= 0 || maxMessageSize > 2048 {
 		return errors.New(" max_message_size_in_mb cannot be more than 2GB (i.e 2048 MB) and has to be a positive number")
 	}
-
+    if err = curationChecks() ;err !=nil {
+    	return err
+	}
 	return validateMeteringChecks()
 }
+func curationChecks() (error) {
+	if GetBool(IsCurationInProgress) {
+		if !common.IsHexAddress(GetString(CurationAddressForValidation)) {
+			return errors.New("A Valid Address needs to be specified to ensure that, only this user can make calls in test during curation process")
+		}
+	}
+	if GetBool(IsCurationInProgress) && GetString(BlockChainNetworkSelected)=="mainnet" {
 
```

Comment:
Typo in message. Lets rephrase it as "Service cannot be curated while set up against Ethereum mainnet"